visualization/dashboard/src/MLIDashboard/Controls/DependencePlot.tsx

```diff
@@ -0,0 +1,361 @@
+import { ComboBox, IComboBox, IComboBoxOption } from "office-ui-fabric-react/lib/ComboBox";
+import * as Plotly from "plotly.js-dist";
+import React from "react";
+import { AccessibleChart, IPlotlyProperty, DefaultSelectionFunctions, PlotlyMode } from "mlchartlib";
+import { mergeStyleSets } from "@uifabric/styling";
+import { IDropdownOption } from "office-ui-fabric-react/lib/Dropdown";
+import { IconButton, Button, DefaultButton } from "office-ui-fabric-react/lib/Button";
+import { Transform } from "plotly.js-dist";
+import { IGenericChartProps, ISelectorConfig, ChartTypes } from "../NewExplanationDashboard";
+import { JointDataset, ColumnCategories } from "../JointDataset";
+import { IExplanationModelMetadata } from "../IExplanationContext";
+import { AxisConfigDialog } from "./AxisConfigurationDialog/AxisConfigDialog";
+import { localization } from "../../Localization/localization";
+import _ from "lodash";
+import { Cohort } from "../Cohort";
+import { Text } from "office-ui-fabric-react";
+import { FabricStyles } from "../FabricStyles";
+
+export interface IDependecePlotProps {
+    chartProps: IGenericChartProps;
+    jointDataset: JointDataset;
+    cohort: Cohort;
+    cohortIndex: number;
+    metadata: IExplanationModelMetadata;
+    onChange: (props: IGenericChartProps) => void;
+}
+
+export class DependencePlot extends React.PureComponent<IDependecePlotProps> {
+    public static basePlotlyProperties: IPlotlyProperty = {
+        config: { displaylogo: false, responsive: true, displayModeBar: false},
+        data: [{}],
+        layout: {
+            dragmode: false,
+            autosize: true,
+            font: {
+                size: 10
+            },
+            margin: {
+                t: 10,
+                l: 10,
+                b: 20,
+                r: 10
+            },
+            hovermode: "closest",
+            showlegend: false,
+            yaxis: {
+                automargin: true,
+                color: FabricStyles.chartAxisColor,
+                tickfont: {
+                    family: "Roboto, Helvetica Neue, sans-serif",
+                    size: 11
+                },
+                zeroline: true,
+                showgrid: true,
+                gridcolor: "#e5e5e5"
+            },
+            xaxis: {
+                automargin: true,
+                color: FabricStyles.chartAxisColor,
+                tickfont: {
+                    family: "Roboto, Helvetica Neue, sans-serif",
```

Comment:
should these repeating font family strings be moved to some common constant for the whole dashboard

Reply:
Yes, I need to discuss with PMs first